### ISSUE TITLE
How to use editorconfig analyzing during VSTS (Azure Devops) build

### ISSUE THREAD

**Issue by WizX20** [MicrosoftDocs/visualstudio-docs#2172]
Can we have a task in a VSTS (Azure DevOps) build that uses the editorconfig to analyze the code? This would be very helpful for pull requests for example.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 72f45eb4-3596-866a-2c16-ee225737d956
* Version Independent ID: cea055e7-fe9b-95c7-6d25-086f182c01d2
* Content: [EditorConfig settings - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2017)
* Content Source: [docs/ide/create-portable-custom-editor-options.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/create-portable-custom-editor-options.md)
* Product: **visual-studio-dev15**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@WizX20 I don't think that is currently possible. See the note block at the end of this section: https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2017#coding-consistency.

**Comment by nschonni** [comment]
If you're running a build, you can invoke https://github.com/jedmao/eclint through Node/NPM to check files for consistency and/or fix the files.
